supabase/client.py

```diff
@@ -80,21 +78,14 @@ def table(self, table_name: str) -> RequestBuilder:
         """
         return self.from_(table_name)
 
-    def from_(self, table_name: str) -> RequestBuilder:
+    def from_(self, table_name: str) -> SyncRequestBuilder:
         """Perform a table operation.
 
         See the `table` method.
         """
-        query_builder = SupabaseQueryBuilder(
-            url=f"{self.rest_url}/{table_name}",
-            headers=self._get_auth_headers(),
-            schema=self.schema,
-            realtime=self.realtime,
-            table=table_name,
-        )
-        return query_builder.from_(table_name)
+        return self.postgrest.from_(table_name)
 
-    def rpc(self, fn: str, params: Dict[Any, Any]) -> Coroutine[Any, Any, Response]:
+    def rpc(self, fn: str, params: Dict[Any, Any]) -> Response:
         """Performs a stored procedure call.
 
         Parameters
```

Comment:
```py
 #     async def remove_subscription_helper(resolve):
    #         try:
    #             await self._close_subscription(subscription)
    #             open_subscriptions = len(self.get_subscriptions())
    #             if not open_subscriptions:
    #                 error = await self.realtime.disconnect()
    #                 if error:
    #                     return {"error": None, "data": { open_subscriptions}}
    #         except Exception as e:
    #             raise e
    #     return remove_subscription_helper(subscription)
```

@J0 why is this code here commented?

Reply:
I think at the time it was written `realtime-py` was not ready for release(still isn't but hopefully soon!) As such we commented it out. I think prior to that we may have had a drop in solution which allowed it to work which is why we left it there but I can't quite remember.